src/addons/manager/addonmanager.h

```diff
@@ -18,6 +18,7 @@ class AddonManager final : public QAbstractListModel {
 
   Q_PROPERTY(
       bool loadCompleted MEMBER m_loadCompleted NOTIFY loadCompletedChanged)
+  Q_PROPERTY(int count READ count NOTIFY countChanged)
```

Comment:
Can we drop the sinal and just expose the prop without it? IIUC the signal is not being used in the tests.

Reply:
A property can be CONSTANT or NOTIFY. If we dont set the signal, the property will not be usable in QML. We are not using it now, but if we implement the property, let's implementing correctly. What do you think?

Reply:
Ah, ok I did not know it was either or. Ok then!